swagger_server/itm/itm_scenario_session.py

```diff
@@ -233,6 +240,87 @@ def _get_sub_directory_names(self, directory):
                 directories.append(item)
         return directories
 
+    def _proper_treatment(self, treatment: str, injury_name: str, location: str) -> bool:
+        # NOTE: Asthmatic, Burns, Forehead Scrape, Ear Bleed are currently untreatable
+
+        """
+            Head Injuries
+            Forehead Scrape: None
+            Face Shrapnel: Airway
+            Ear Bleed: None
+
+            Neck Injuries
+            Neck Puncture: Hemostatic gauze
+
+            Hand Injuries
+            Wrist Amputation: Tourniquet
+            Palm Laceration: Pressure bandage
+
+            Arm Injuries
+            Forearm Laceration: Pressure bandage
+            Bicep Puncture: Tourniquet
+            Shoulder Puncture: Hemostatic gauze
+
+            Chest Injuries
+            Asthmatic: None
+            Chest Collapse: Decompression Needle
+
+            Stomach Injuries
+            Stomach Puncture: Hemostatic gauze
+            Side Puncture: Hemostatic gauze
+
+            Leg Injuries
+            Thigh Puncture: Tourniquet
+            Thigh Laceration: Tourniquet
+            Shin Amputation: Tourniquet
+            Calf Laceration: Pressure bandage
+        """
+        match injury_name:
+            case 'Amputation':
+                return treatment == 'Tourniquet'
+            case 'Chest Collapse':
+                return treatment == 'Decompression Needle'
+            case 'Laceration':
+                if 'thigh' in location:
+                    return treatment == 'Tourniquet'
+                else:
+                    return treatment == 'Hemostatic gauze'
+            case 'Puncture':
+                if 'bicep' in location or 'thigh' in location:
+                    return treatment == 'Tourniquet'
+                else:
+                    return treatment == 'Hemostatic gauze'
+            case 'Shrapnel':
+                return treatment == 'Nasopharyngeal airway'
+            case _:
+                return False
+
+
+    def _reveal_injuries(self, source: Casualty, target: Casualty):
```

Comment:
May not be necessary to change but I got confused at first looking at the nested for each loops and the return statement at the beginning. This should keep the same code logic but make it a little more concise:
`def _reveal_injuries(self, source: Casualty, target: Casualty):
    if not target.assessed:
        for source_injury in source.injuries:
            if source_injury.name == 'Burn' and not any(target_injury.name == 'Burn' for target_injury in target.injuries):
                target.injuries.append(source_injury)`

Reply:
Done, thank you.  I like the initial return statement for readability and to prevent extra indenting, but I did the rest, and added a `hidden_injury_types` list instead of hardcoding `Burn` in several places.  In the September scenarios, the only hidden injury type is Burns.